app/nav/CmsNavbar.tsx

```diff
@@ -0,0 +1,58 @@
+import {
```

Comment:
I don't think this file naming scheme fits well

Reply:
What would you suggest?